### ISSUE TITLE
Does btcrecover support bitcoin cores that are in chinese?

### ISSUE THREAD

**Issue by kost123** [gurnec/btcrecover#54]

**Comment by gurnec** [comment]
Yes it does, if you use [the `--utf8` command-line option](https://github.com/gurnec/btcrecover/blob/master/TUTORIAL.md#unicode-support) and save the tokens.txt file in UTF-8 format, btcrecover will be able to use it. However, it's unfortunately not as well tested as plain ASCII and is more likely to have bugs....

**Comment by btcexchanging** [comment]
if you haven't recovered your wallet, i can help you recover your wallet.
I have recovered most of the common wallets and recovery phrases for many users
you can reach me here at my telegram account:  @walletrecover
